source/portable/NetworkInterface/xilinx_ultrascale/x_emacpsif_hw.c

```diff
@@ -173,7 +181,7 @@ static void emacps_handle_error( void * arg,
                 if( ( ErrorWord & XEMACPS_RXSR_HRESPNOK_MASK ) != 0 )
                 {
                     last_err_msg = "Receive DMA error";
-                    xNetworkInterfaceInitialise();
+                    vInitialiseOnIndex( xEMACIndex );
```

Comment:
I am not sure whether these are any good as `vInitialiseOnIndex()` cannot be run again when it once ended up in `xEMAC_Fatal` or `xEMAC_Ready`.